integration-tests/kubernetes/maven-invoker-way/src/it/kubernetes-docker-deployment-additional-roles/verify.groovy

```diff
@@ -0,0 +1,40 @@
+import io.dekorate.utils.Serialization
```

Comment:
For the biggest part of the kubernetes extension we use `standard` tests. We only add `invoker` tests to test aspects that are specific to build tool (e.g. passing options to the maven build etc).

Reply:
+1. Let's try and not add invoker tests unless absolutely necessary